packages/common/src/components/AutocompleteInput.jsx

```diff
@@ -0,0 +1,53 @@
+import { useState, Fragment } from 'react'
+import { Combobox } from '@headlessui/react'
+import { XIcon } from '@heroicons/react/outline'
+
+export default function AutocompleteInput(props) {
+  const { items, selectedItems, setSelectedItems } = props
+  const [query, setQuery] = useState('')
+
+  function removeSelectedItem(itemId) {
+      const newSelectedItems = selectedItems.filter(item => item.id != itemId)
+      setSelectedItems(newSelectedItems)
+  }
+
+  const filteredItems =
+    query === ''
+      ? items
+      : items.filter((item) => {
+          return item.name.toLowerCase().includes(query.toLowerCase())
+        })
+
+  return (
+    <Combobox value={selectedItems} onChange={setSelectedItems} multiple>
+      <Combobox.Input
```

Comment:
We need to allow for custom values so we can still add members by id only [0]


[0] https://headlessui.dev/react/combobox#allowing-custom-values